### PR TITLE
Allow PlainTextLoader to select encoding and defaults to UTF-8

### PR DESCRIPTION
**Pull request by xuhdev**
This should resolve all Windows failures

### CODE REVIEW

**Comment by xuhdev** [comment]
I'll check on Windows test checkboxes as required once you approval/merge

**Review by edwardleardi — APPROVED** [review]
Nice work, congrats on getting all the tests to pass on windows. That was way more tedious than I thought it would be when we first added the windows jobs.

**Comment by xuhdev** [comment]
Me, too, that's why the Windows issue was transformed to be an umbrella

**Pull request merged**
